### PR TITLE
Added features of the CORTX Motr

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added features of the CORTX Motr

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,6 +24,20 @@ Motr
 At the core of `CORTX <https://github.com/Seagate/cortx>`_ lies Motr.  Motr is a distributed object storage system, targeting `mass capacity storage <https://www.seagate.com/products/storage/object-storage-software/>`_
 configurations. To ensure the most efficient storage utilization, Motr interacts directly with block devices (i.e. it does not _layer_ on a local file system).  The Motr design was heavily influenced by the Lustre file system, NFSv4 and database technology. It must be noted that traditional file system properties (hierarchical directory namespace, strong POSIX consistency guarantees, &c.) are no longer desirable or achievable at mass capacity. Instead, Motr is a more general storage system that provides an optional file system interface. This allows wider range of deployments, including cloud.
 
+Following are the features of the Motr submodule:
```
Comment (huanghua78) [inline README.rst:27]: Please remove the word "submodule".

At first glimpse, I was confused it with the "git submodule".

Can we say: 
* Cortx Motr
* The Motr subsystem
* The Motr storage system
* The Motr object storage system
* The Motr
?

File: README.rst [hunk 2]
```diff
@@ -24,6 +24,20 @@ Motr
...
+
+- Scalable:
```
Comment (huanghua78) [inline README.rst:29]: Only this "Scalable" is shown as _ITALIC_.
Why?
Make all them _ITALIC_ and **BOLD**, or make them the same.

File: README.rst (hunk 1 above)
Comment (Saumya-Sunder) [reply README.rst:27]: CORTX Motr is better.

File: README.rst (hunk 2 above)
Comment (Saumya-Sunder) [reply README.rst:29]: Sure. Changing

**Review by huanghua78 — APPROVED** [review]

**Pull request merged**